docs/tutorials/ad-hoc-simulation/planning-simulation.md

```diff
@@ -17,53 +17,77 @@
 
 ## How to run a planning simulation
 
-1. Launch Autoware.
+### Launch Autoware
 
-   ```sh
-   source ~/autoware/install/setup.bash
-   ros2 launch autoware_launch planning_simulator.launch.xml map_path:=$HOME/Downloads/sample-map-planning vehicle_model:=sample_vehicle sensor_model:=sample_sensor_kit
-   ```
+```sh
+source ~/autoware/install/setup.bash
+ros2 launch autoware_launch planning_simulator.launch.xml map_path:=$HOME/Downloads/sample-map-planning vehicle_model:=sample_vehicle sensor_model:=sample_sensor_kit
+```
 
-   Note that you cannot use `~` instead of `$HOME` here.
+Note that you cannot use `~` instead of `$HOME` here.
 
-   ![after-autoware-launch](images/planning/lane-following/after-autoware-launch.png)
+![after-autoware-launch](images/planning/lane-following/after-autoware-launch.png)
 
-2. Set an initial pose for the ego vehicle.
+### Open Autoware State Panel
 
-   - a) Click the `2D Pose estimate` button in the toolbar, or hit the `P` key
-   - b) In the 3D View pane, click and hold the left-mouse button, and then drag to set the direction for the initial pose.
+This panel is useful when we start planning simulation. Click `Panels -> Add new panel`, select `AutowareStatePanel`, and then press `OK`.
 
-   ![set-initial-pose](images/planning/lane-following/set-initial-pose.png)
+![after-autoware-launch](images/planning/lane-following/open-autoware-state-panel.png)
 
-3. Set a goal pose for the ego vehicle.
+### Set an initial pose for the ego vehicle
 
-   - a) Click the `2D Nav Goal` button in the toolbar, or hit the `G` key
-   - b) In the 3D View pane, click and hold the left-mouse button, and then drag to set the direction for the goal pose.
+1. Click the `2D Pose estimate` button in the toolbar, or hit the `P` key.
+2. In the 3D View pane, click and hold the left-mouse button, and then drag to set the direction for the initial pose. The car should be placed in the right direction according to the lane(you can check the arrow on the lane). Now you will see a car on the Rviz.
 
-   ![set-goal-pose](images/planning/lane-following/set-goal-pose.png)
+![set-initial-pose](images/planning/lane-following/set-initial-pose.png)
 
-4. To place a dummy object such as a pedestrian, click the `2D Dummy Pedestrian` button in the toolbar.
+### Set a goal pose for the ego vehicle
 
-   ![set-dummy-pedestrian](images/planning/lane-following/set-dummy-pedestrian.png)
+1. Click the `2D Nav Goal` button in the toolbar, or hit the `G` key
+2. In the 3D View pane, click and hold the left-mouse button, and then drag to set the direction for the goal pose. Keep in mind to set the goal pose aligned with the lane direction. Otherwise the path will not be planned. Now you will see a planned path.
```

Comment:
Changed the warning about goal pose alignment to be a warning using the mkdocs admonition code block.

```suggestion
2. In the 3D View pane, click and hold the left-mouse button, and then drag to set the direction for the goal pose.  If done correctly, you will see a planned path from initial pose to goal pose.

!!! warning

    Remember to set the goal pose to be aligned with the lane direction, otherwise the path will not be planned! 
```